src/core/src/op/random_uniform.cpp

```diff
@@ -92,6 +92,14 @@ bool RandomUniform::evaluate(TensorVector& outputs, const TensorVector& inputs)
     const auto& t_out = get_out_type();
     OPENVINO_ASSERT(validate::out_et(t_out), "Unsupported type of RandomUniform: " + t_out.get_type_name());
 
+    auto in0_data = inputs[0].data();
+    Shape output_shape;
+    for (int i = 0; i < inputs[0].get_size(); i++) {
+        output_shape.push_back(out_dims[i]);
+    }
+
+    outputs[0].set_shape(output_shape);
```

Comment:
Can the `out_shape` variable (the result of `shape_infer` L89) be reused here instead of copying from `out_dims` vector?
```suggestion
    outputs[0].set_shape(out_shape);
```

Reply:
Right, thanks